numpy/core/_add_newdocs.py

```diff
@@ -4424,6 +4424,11 @@
         since the former invokes ``dtype(None)`` which is an alias for
         ``dtype('float_')``.
 
+    .. versionchanged:: 1.21.1
```

Comment:
This will be part of the next release, which is 1.23, or did I misunderstand?
```suggestion
    .. versionchanged:: 1.23.0
```